themes/default/layouts/partials/registry/header.html

```diff
@@ -70,37 +70,9 @@
 
 <header class="header-container registry-hero">
     <div class="hide-on-pinned">
-        <div class="dot-background-container">
-            <div class="dot-background no-animate"></div>
-        </div>
-        <div class="dot-overlay"></div>
-    </div>
-
-<div class="search-header container flex">
-    <div class="header-pinned-container hidden md:flex xl:w-3/12 px-4 lg:px-8 xl:px-5">
-        <div class="header-hero-title pr-3">
-            <div class="flex items-center">
-                <a href="{{ relref . " /registry" }}">
-                    <i class="fas fa-archive title-icon"></i>
-                </a>
-                <a href="{{ relref . " /registry" }}">
-                    {{/* Only use an h1 on the index page. */}}
-                    {{ if eq .Title "Pulumi Registry"}}
-                    <h1 class="title-text-content no-anchor">Pulumi Registry</h1>
-                    {{ else }}
-                    <span class="title-text-content">Pulumi Registry</span>
-                    {{ end }}
-                </a>
-            </div>
-        </div>
-        <div class="header-pinned items-center transition-all hidden">
-            <img class="h-6 mr-12" src="/images/logo/logo-on-white.svg" alt="Pulumi logo" />
-            <div class="get-started-pinned-search-container">
-                <a class="get-started-pinned-search-button" data-track="get-started-pinned-search" 
-                href="{{ relref . "/docs/get-started" }}">Get Started</a>
-            </div>
-        </div>
     </div>
+    {{ .Scratch.Set "mode" "registry" }}
+    {{ partial "docs/search.html" . }}
```

Comment:
this is now using a shared partial between docs and registry, which is defined in the hugo PR. Note, the specific scripts to work with swift-type setting between the two have remained here, so the search results logic themselves will still be the same as before. It is just the search bar component styling that is now shared.